### ISSUE TITLE
chore: format the `configExample.json`

### ISSUE THREAD

**Issue by AceLikesGhosts** [iSlammedMyKindle/kindle-twitch-oauth#4]
not needed but it hurts me mentally to see `configExample.json` to have unformatted json

**Comment by iSlammedMyKindle** [comment]
It's unformatted? I'm blind, can you tell me what you mean by that?
![image](https://github.com/iSlammedMyKindle/kindle-twitch-oauth/assets/108752129/9fd2e176-6be2-4538-821a-4eae29258501)

**Comment by AceLikesGhosts** [comment]
honestly dont remember, i believe i was viewing it on my phone and it murdered the formatting; looking at it now and it's not unformated. don't know tbh, my bad 😭

**Comment by iSlammedMyKindle** [comment]
no problem XP

**Comment by AceLikesGhosts** [comment]
> no problem XP

so like unblock me on discord? 🥺

**Comment by iSlammedMyKindle** [comment]
That's not how it works @AceLikesGhosts 

I'm not going to discuss this up on a public repository. You know very well what you did, but to respect anyone potentially employing you in the future, I'm going to end the discussion here.

If there are any other messages like this on my repos, I will delete them. I hope you understand.

**Comment by iSlammedMyKindle** [comment]
Actually, I gave this some much deeper though, @AceLikesGhosts I think it would be wise for me to cut ties all the way.

I appreciate code contributions regardless of who stops by, but in our case this is one that is tied to the phrase "don't the hand that feeds ya"

This is a mistake I made and probably brought hopes up to being unblocked. My first instinct should have been to reject code changes if my goal was to cut ties in general.

Ace I want you to succeed professionally, but that means we need to be rid of anything involving us in any place. I'm not here to communicate with you anymore at this point. If you want a fresh start, I would suggest looking for a fresh community. There are a lot of other code repositories besides mine.

I wish you the best of luck professionally, this is goodbye.
